src/main/java/org/junitpioneer/jupiter/cartesian/CartesianTest.java

```diff
@@ -310,4 +310,13 @@ void validate(CartesianTest.Enum enumSource, Set<? extends java.lang.Enum<?>> co
 
 	}
 
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target({ ElementType.METHOD, ElementType.ANNOTATION_TYPE })
+	@CartesianArgumentsSource(CartesianFactoryArgumentsProvider.class)
+	@interface Factory {
```

Comment:
Should we call this `MethodSource` in line with Jupiter or is there a good reason to have a different name (e.g. to avoid collisions)?

Reply:
No, we should not. For several reasons:
 - In `@CartesianProductTest` this was a property called `factory`.
 - It avoids collision, but more importantly it avoids confusion. This annotation is only for `@CartesianTest`.
 - We decided to use short names for simplicity. Even though this annotation doesn't go on a parameter, I'd still keep it short.
 - I don't want to change it. 😭

Reply:
I have to think about it. If I feel we need to change it, I can do the work. 😉